RFC/src/RFC-0202_TariScriptOpcodes.md

```diff
@@ -325,6 +325,23 @@ push 0.
 Identical to [`CheckSig`](#checksigmsg), except that nothing is pushed to the stack if the signature is valid, and the
 operation fails with `VERIFY_FAILED` if the signature is invalid.
 
+##### CheckMultiSig(Msg)
+
+Pop the number $n$ indicating the number of allowed spenders, and then pop $n$ spender public keys off the
```

Comment:
I don't think `spender` is the correct term here. You can use MultiSig for a number of use cases so I think it would be better to call them `signers`

Reply:
Thanks will update